jobsAdmin/init.go

```diff
@@ -476,6 +477,30 @@ func GetJobSummary(jobID common.JobID) common.ListJobSummaryResponse {
 	}
 	part0PlanStatus := part0.Plan().JobStatus()
 
+	// This is added to let FE to continue fetching the Job Progress Summary
+	// in case of resume. In case of resume, the Job is already completely
+	// ordered so the progress summary should be fetched until all job parts
+	// are iterated and have been scheduled
+	js.CompleteJobOrdered = js.CompleteJobOrdered || jm.AllTransfersScheduled()
+
+	if part0PlanStatus != common.EJobStatus.Cancelled() && part0PlanStatus != common.EJobStatus.Failed() &&
+		(js.CompleteJobOrdered) && (part0PlanStatus.IsJobDone()) {
+		/*
+		 * Plan file (hence part0PlanStatus.IsJobDone) is updated inline by jm.reportJobPartDoneHandler()
```

Comment:
Discussed with @nitin-deamon  and understood that this is race between jobPartDone handler and statusManager. This should be fixed by resolving that race. I've fixed the race in a simpler way in #1814 . Please revert this part of PR i.e. anything related to draining messages from statusMgr

Reply:
Did a git revert for the same.